miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -29,9 +29,11 @@ abstract class MiniAppMessageBridge {
     private lateinit var miniAppInfo: MiniAppInfo
     private lateinit var activity: Activity
     private lateinit var userInfoBridgeDispatcher: UserInfoBridgeDispatcher
-    internal lateinit var screenBridgeDispatcher: ScreenBridgeDispatcher
     private val adBridgeDispatcher = AdBridgeDispatcher()
 
+    private lateinit var screenBridgeDispatcher: ScreenBridgeDispatcher
+    private var allowScreenOrientation = false
```

Comment:
I think, if we directly assign `false` inside `ScreenBridgeDispatcher`'s constructor, then we can remove this line:

e.g.

```
internal class ScreenBridgeDispatcher(
    ....
    var allowScreenOrientation: Boolean = false
) 

abstract class MiniAppMessageBridge {
   ...
    private lateinit var screenBridgeDispatcher: ScreenBridgeDispatcher

    internal fun init(
        ...
    ) {
        ...
        this.screenBridgeDispatcher = ScreenBridgeDispatcher(activity, bridgeExecutor)
        ...
   }

  fun allowScreenOrientation(isAllowed: Boolean) {
        if (this::screenBridgeDispatcher.isInitialized)
            screenBridgeDispatcher.allowScreenOrientation = isAllowed
   }
}
```

Reply:
yeah, let me update it.

Reply:
ah I check it again and see that we cannot go that way. The `allowScreenOrientation(isAllowed: Boolean)` will not change the state of `ScreenBridgeDispatcher` after it is initialized.